.dotnet/src/Custom/Batch/BatchOperationToken.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.ClientModel;
+using System.Diagnostics;
+using System.IO;
+using System.Text.Json;
+
+#nullable enable
+
+namespace OpenAI.Batch;
+
+internal class BatchOperationToken : ContinuationToken
+{
+    public BatchOperationToken(string batchId)
+    {
+        BatchId = batchId;
+    }
+
+    public string BatchId { get; }
+
+    public override BinaryData ToBytes()
+    {
+        using MemoryStream stream = new();
+        using Utf8JsonWriter writer = new(stream);
+        writer.WriteStartObject();
+
+        writer.WriteString("batchId", BatchId);
```

Comment:
can we pass UTF8 string?

Reply:
Yes!

Reply:
Saving this to come back to in a separate PR